### PR TITLE
fix: page not found for apisix docker and apisix helm chart

### LINKED ISSUE #281: page not found for Apisix Docker and Apisix Helm Chart
**Actual Behaviour**

Screenshot (74)

**Expected Behaviour**
Page has to  be accessed 

**Steps to reproduce it**
go to docusaurus.config.js
add some codes like https://github.com/apache/apisix-website/blob/master/website/docusaurus.config.js

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by KishaniKandasamy**
Fixes: #281

Changes:

update docusaurus.config.js

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -192,6 +192,42 @@ module.exports = {
         },
       },
     ],
+    [
+      "@docusaurus/plugin-content-docs",
```
Comment (juzhiyuan) [inline website/docusaurus.config.js:175]: wow thanks, this will need @qier222's help to review :)

**Comment by juzhiyuan** [comment]
Hi @KishaniKandasamy, I noticed that we use `downloads` fields to render both `Doc Selector` on the navbar and `Downloads` part, that's not a good way, due to we don't need `docker` on the Download page.

Because we removed them from Downloads field, then there have some conflicts, would you like to add a new field like `docs` on the config file? and render the navbar from it.

**Comment by KishaniKandasamy** [comment]
> Hi @KishaniKandasamy, I noticed that we use `downloads` fields to render both `Doc Selector` on the navbar and `Downloads` part, that's not a good way, due to we don't need `docker` on the Download page.
> 
> Because we removed them from Downloads field, then there have some conflicts, would you like to add a new field like `docs` on the config file? and render the navbar from it.

yes I will  👍🏻

**Comment by KishaniKandasamy** [comment]
> Hi @KishaniKandasamy, I noticed that we use `downloads` fields to render both `Doc Selector` on the navbar and `Downloads` part, that's not a good way, due to we don't need `docker` on the Download page.
> 
> Because we removed them from Downloads field, then there have some conflicts, would you like to add a new field like `docs` on the config file? and render the navbar from it.

Hi Zhiyuan, then  **"@docusaurus/plugin-content-docs**" for helm chart and docker??

**Comment by juzhiyuan** [comment]
> > Hi @KishaniKandasamy, I noticed that we use `downloads` fields to render both `Doc Selector` on the navbar and `Downloads` part, that's not a good way, due to we don't need `docker` on the Download page.
> > Because we removed them from Downloads field, then there have some conflicts, would you like to add a new field like `docs` on the config file? and render the navbar from it.
> 
> Hi Zhiyuan, then **"@docusaurus/plugin-content-docs**" for helm chart and docker??

We should keep them :) Because Docusarus will render docs.

**Review by juzhiyuan — APPROVED** [review]

File: website/blog/2021-03-27-traffic-split-in-apache-apisix-ingress-controller.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+Requests whose Host is "foo.org", URI path prefix is "/get" will be separated into two parts:
+
++ The id parameter which value is 3, 13, 23 or 33 will hit rule2, and forward to foo-canary;
+
++ Others will hit rule1 and route to foo.
+
+## Summary
+
+Traffic split (Canary release) in Ingress Nginx supports weight-based scheme and header rule-based one, but it relies on annotations, which semantic is weak; The Kong way only supports to configure canary release by weight, the scenarios are somewhat narrow, and the configuring is complicated (you need to configure several resources); In contrast, traffic split in Apache APISIX Ingress Controller is flexible and easy to configure, it works well for both the weight-based and rule-based traffic split scheme.
```
Comment (jbampton) [inline website/blog/2021-03-27-traffic-split-in-apache-apisix-ingress-controller.md:177]:
```suggestion
Traffic split (Canary release) in Ingress Nginx supports weight-based scheme and header rule-based one, but it relies on annotations, which semantically is weak; The Kong way only supports to configure canary release by weight, the scenarios are somewhat narrow, and the configuring is complicated (you need to configure several resources); In contrast, traffic split in Apache APISIX Ingress Controller is flexible and easy to configure, it works well for both the weight-based and rule-based traffic split scheme.
```

https://www.thefreedictionary.com/semantically

**Review by membphis — APPROVED** [review]

**Comment by juzhiyuan** [comment]
Hi @jbampton, after I merged this PR, I noticed that you have a suggestion change  🤣🤣

**Pull request merged**